src/buttons.c

```diff
@@ -0,0 +1,9 @@
+#include "buttons.h"
+
+/**
+ * @brief Function to read user button.
+ */
```

Comment:
Maybe we could try to write similar names for the functions... (button_left_read currenlty on bulebule ot read_user_button on Meiga)?

Reply:
Fixed in a688768c8d781f2ffbbcd9e274f81c2f493bd2fa.

Reply:
great